aviatrix/data_source_aviatrix_gateway.go

```diff
@@ -4,6 +4,8 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/terraform-providers/terraform-provider-aviatrix/goaviatrix/cloud"
```

Comment:
There is no need to import "*/goaviatrix/cloud" since "/goaviatrix" is imported as a whole, right?

Reply:
We have to do it like this since `cloud` is still it's own package it's just located inside goaviatrix/ directory
```
package cloud

// Cloud provider ids
const (
	AWS    = 1
	GCP    = 4
	AZURE  = 8
	OCI    = 16
	AWSGOV = 256
)
```